web/js/index.js

```diff
@@ -263,12 +263,19 @@ $elemSelector.addEventListener('change', async (evt) => {
             document.body.style.cursor = 'progress';
             const resp = await fetch(`/api/gltf${evt.target.options[event.target.selectedIndex].getAttribute('href')}`);
             const json = await resp.json();
-            poll(5, () => fetch(`/api/gltf/${json.id}`), (resp) => resp.status !== 202, (respJson) => {
+            poll(5, () => fetch(`/api/gltf/${json.id}`), (resp) => resp.status !== 202, (resp) => {
+                let respJson;
+                try {
+                    respJson = JSON.parse(resp);
+                } catch (error) {
+                    displayError('There was an error translating the model to GLTF: ' + error);
```

Comment:
Can you make the error message a constant some where & reuse it in the other place.
`There was an error translating the model to GLTF:`

Reply:
I just realized that the outer catch block will catch JSON.parse() errors (if any are thrown), so the try-catch block on line 268 is not needed.
Hence the error message `There was an error translating the model to GLTF:` will not need to be reused.